diff --git a/diff/src/Controller/PluginRevisionController.php b/diff/src/Controller/PluginRevisionController.php
index 6205ae1..da1a998 100644
--- a/src/Controller/PluginRevisionController.php
+++ b/src/Controller/PluginRevisionController.php
@@ -255,7 +255,7 @@ class PluginRevisionController extends ControllerBase {
       }
       if ($i != 0) {
         // Build the left link.
-        $left_link = Link::fromTextAndUrl($this->t('Previous change'), $this->diffRoute($entity, $revision_ids[$i - 1], $left_revision_id, $filter, $layout_options))->toString();
+        $left_link = Link::fromTextAndUrl($this->t('Previous revision'), $this->diffRoute($entity, $revision_ids[$i - 1], $left_revision_id, $filter, $layout_options))->toString();
       }
       $element['left'] = [
         '#type' => 'markup',
@@ -270,7 +270,7 @@ class PluginRevisionController extends ControllerBase {
       }
       if ($revisions_count != $i && $revision_ids[$i - 1] != $revision_ids[$revisions_count - 1]) {
         // Build the right link.
-        $right_link = Link::fromTextAndUrl($this->t('Next change'), $this->diffRoute($entity, $right_revision_id, $revision_ids[$i], $filter, $layout_options))->toString();
+        $right_link = Link::fromTextAndUrl($this->t('Next revision'), $this->diffRoute($entity, $right_revision_id, $revision_ids[$i], $filter, $layout_options))->toString();
       }
       $element['right'] = [
         '#type' => 'markup',
diff --git a/diff/src/Form/RevisionOverviewForm.php b/diff/src/Form/RevisionOverviewForm.php
index 45c2c66..932fae7 100755
--- a/src/Form/RevisionOverviewForm.php
+++ b/src/Form/RevisionOverviewForm.php
@@ -175,8 +175,8 @@ class RevisionOverviewForm extends FormBase {
     // Allow comparisons only if there are 2 or more revisions.
     if ($revision_count > 1) {
       $table_header += array(
-        'select_column_one' => '',
-        'select_column_two' => '',
+        'select_column_one' => 'Compare this',
+        'select_column_two' => 'with this',
       );
     }
     $table_header['operations'] = $this->t('Operations');
